frameworks/Ruby/rails/app/controllers/hello_world_controller.rb

```diff
@@ -24,9 +24,9 @@ def query
   end
   
   def fortune
-    @fortunes = Fortune.all
+    @fortunes = Fortune.order(:message)
     @fortunes << Fortune.new(:id => 0, :message => "Additional fortune added at request time.")
-    @fortunes = @fortunes.sort_by { |x| x.message }
+    @fortunes.sort_by!(&:message)
```

Comment:
My hypothesis was that it would make the Ruby-sorting faster, but it looks like using ORDER BY is actually prohibited by the rules anyway, so I'll remove it.